assets/queries/cloudFormation/iam_access_keys_active_for_root/query.rego

```diff
@@ -0,0 +1,18 @@
+package Cx
+
+CxPolicy [ result ] {
+  accessKey := input.document[i].Resources[name]
+  accessKey.Type == "AWS::IAM::AccessKey"
+
+  userRef := accessKey.Properties.UserName
+
+  contains(lower(userRef),"root")
+
+	result := {
+                "documentId": 		input.document[i].id,
+                "searchKey": 	    sprintf("Resources.%s.Properties.UserName", [name]),
+                "issueType":		"IncorrectValue",  
+                "keyExpectedValue": sprintf("'Resources.%s.Properties.UserName' is not asssociated to root account.",[name]),
+                "keyActualValue": sprintf("'Resources.%s.Properties.UserName' is asssociated to root account.",[name])
+              }
+}
```

Comment:
[As a good POSIX practice](https://unix.stackexchange.com/a/18789) add end-of-file newline